templates/telemetry/rocket_android_events_v1/view.sql

```diff
@@ -233,47 +234,7 @@ WHERE
-      UNION ALL SELECT 'to_build' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'to_build') AS value
-      UNION ALL SELECT 'action' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'action') AS value
-      UNION ALL SELECT 'finish' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'finish') AS value
-      UNION ALL SELECT 'page' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'page') AS value
-      UNION ALL SELECT 'message_id' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'message_id') AS value
-      UNION ALL SELECT 'link' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'link') AS value
-      UNION ALL SELECT 'background' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'background') AS value
-      UNION ALL SELECT 'primary' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'primary') AS value
-      UNION ALL SELECT 'vertical' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'vertical') AS value
-      UNION ALL SELECT 'component_id' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'component_id') AS value
-      UNION ALL SELECT 'feed' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'feed') AS value
-      UNION ALL SELECT 'subcategory_id' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'subcategory_id') AS value
-      UNION ALL SELECT 'version_id' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'version_id') AS value
-      UNION ALL SELECT 'app_link' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'app_link') AS value
-      UNION ALL SELECT 'session_time' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'session_time') AS value
-      UNION ALL SELECT 'show_keyboard' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'show_keyboard') AS value
-      UNION ALL SELECT 'url_counts' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'url_counts') AS value
-      UNION ALL SELECT 'impression' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'impression') AS value
-      UNION ALL SELECT 'loadtime' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'loadtime') AS value
-      UNION ALL SELECT 'audience_name' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'audience_name') AS value
-      UNION ALL SELECT 'finished' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'finished') AS value
-      UNION ALL SELECT 'task' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'task') AS value
-      UNION ALL SELECT 'item_name' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'item_name') AS value
+      SELECT 'event_value' AS key, event_value AS value
```

Comment:
* generic `event_value` is added here for new events
* event properties that were explicitly extracted here are removed because they're all automatically pulld into `event_props_1` couple of lines above

Reply:
I don't understand this. Can you elaborate more?

Reply:
>     * generic `event_value` is added here for new events

Previously `event_value` was used for distinguishing and naming events (big `all_events` section above). We are keeping this for old events. But we'll also pull `event_value` to a parameter which is for new event types, in order to enable filtering in Amplitude.

>     * event properties that were explicitly extracted here are removed because they're all automatically pulled into `event_props_1` couple of lines above

This is implementation detail - since whole `event_map_values` map is inserted into event_parameters, these explicit extract statements (e.g. `UNION ALL SELECT 'item_name' AS key, `moz-fx-data-derived-datasets.udf.get_key`(event_map_values, 'item_name') AS value`) are not necessary.